modules/rte_mis_school/src/Batch/SchoolUdiseCodeBatch.php

```diff
@@ -274,11 +276,19 @@ public static function finishedCallback(bool $success, array $results, array $op
           $renderer = \Drupal::service('renderer');
           $title = $renderer->render($markup);
           $final_error_messages[] = $list_items;
-          \Drupal::messenger()->addWarning($title);
-          \Drupal::messenger()->addWarning($renderer->render($list_items));
+
         }
 
+        $url = Url::fromUri('internal:/failed-bulk-udise-upload');
+        $link = Link::fromTextAndUrl('details', $url)->toString();
+
+        $markup = [
+          '#type' => 'markup',
+          '#markup' => "Import failed. Here are the $link",
```

Comment:
Change the message to `Bulk Upload Failed. Click here to see the list of errors.`